.gitignore

```diff
@@ -0,0 +1,136 @@
+# Logs
```

Comment:
I don't think many of these are required for a Freshworks app. You can generate this file with the `fdk generate` file and it will provide all the relevant details.